json-schema-bundler-cli.mjs

```diff
@@ -73,8 +77,15 @@ try {
     process.exit(1);
 }
 
+let result;
 if (argv.y) {
-    console.log(YAML.dump(schema, argv.p ? undefined : { flowLevel: 3 }));
+    result = YAML.dump(schema, argv.p ? undefined : { flowLevel: 3 });
 } else {
-    console.log(JSON.stringify(schema, undefined, argv.p ? 2 : undefined));
+    result = JSON.stringify(schema, undefined, argv.p ? 2 : undefined);
+}
+
+if (argv.o === '') {
```

Comment:
If the option "-o" has not been passed it will lead to an error:
```
node:fs:2405
      path = getValidatedPath(path);
             ^

TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received undefined
    at Object.writeFileSync (node:fs:2405:14)
    at file:///json-schema-bundler/json-schema-bundler-cli.mjs:90:8 {
  code: 'ERR_INVALID_ARG_TYPE'
}